CHANGES.md

```diff
@@ -1,3 +1,33 @@
+## edge-20.2.3
+
+This release introduces the first optional add-on `tracing`, added through the
+new add-on model!
+
+The existing optional `tracing` components Jaeger and OpenCensus can now be
+installed as add-on components.
+
+There will be more information to come about the new add-on model, but please
+refer to the [issue](https://github.com/linkerd/linkerd2/issues/3794) for how
```

Comment:
The getting started info is in the PR (https://github.com/linkerd/linkerd2/pull/3955), not the issue.

Reply:
@ihcsim I'll link directly to the [process](https://github.com/linkerd/linkerd2/pull/3955#issuecomment-576288151) comment

Reply:
It's slightly different; the process is about how to introduce a new add-on. The PR talks about how to enable Jaeger and OpenCensus during installation. I think enabling the tracing add-on will be more interesting to the user, than the process to introduce a new add-on.

Reply:
@ihcsim Ah okay yea my wording "how to get started" was written with the meaning of "how to get started in adding add-ons". For this initial release, it's probably good to just explain how to add the tracing add-on, and allow for process documentation at a later point.